parsl/executors/ipp.py

```diff
@@ -156,8 +158,8 @@ def compose_launch_cmd(self, filepath, engine_dir, container_image):
 {1}
 EOF
 
-mkdir -p '.ipengine_logs'
-ipengine --file=ipengine.{uid}.json {debug_option} >> .ipengine_logs/$JOBNAME.log 2>&1
+mkdir -p 'engine_logs'
```

Comment:
Can you confirm that the engine_dir is actually used in this launch script ?

Reply:
Yes, it is the directory that the batch jobs changes to before creating the `engine_logs` directory.